packages/client/components/TaskEditor/withEmojis.tsx

```diff
@@ -29,10 +31,22 @@ const withEmojis = (ComposedComponent) => {
 
     menuRef = React.createRef<any>()
 
-    menuItemClickFactory = (emoji, editorState) => (e) => {
-      e.preventDefault()
+    static getDerivedStateFromProps(nextProps, prevState) {
+      const {editorState} = nextProps
+      return {
+        // clicking on a menu will cause the editorStateSelection to lose focus, so we persist the last state before that point
+        focusedEditorState: editorState.getSelection().getHasFocus()
+          ? editorState
+          : prevState.focusedEditorState
+      }
+    }
```

Comment:
I'm not sure if this exact logic is necessary, but I've copied this from `withEmojis` to maintain identical behavior